index.html

```diff
@@ -19,7 +19,7 @@
 
         </header>
 
-        <section class="background-image">
+        <div class="background-image">
```

Comment:
You should change the tag  back to the section tag to align with use of good semantics

Reply:
- [ ] Kindly use nice words to the student when asking for a change in their code. In this case, it would be nice if you use words like: 👍 
- Kindly make this `div` a section tag instead of a div as that will make your code semantic. Or 
- It would be nice if you use a section tag here as that will make your code semantic